### PR TITLE
Update analysis based on co-authors comments

### PR DESCRIPTION
**Pull request by ajlee21**
In preparation for this paper submission, major changes in this PR include:

1. Update the KEGG annotations used. Currently we are using the ones formatted by Jie from her Tan et al. publication, however these KEGG annotations are fairly old now. (added `3_core_core_analysis/0_get_kegg_pathways.ipynb` and updated `3_core_core_analysis/3_find_KEGG_associations.ipynb` and `3_core_core_analysis/5_KEGG_enrichment_of_stable_genes.ipynb`)
2. Performed analysis to compare size of operon to transcriptional stability of genes (added `3_core_core_analysis/8_stability_vs_operon_size.ipynb`)

Other minor changes that don't need review:
1. Printed out samples that had high PAO1 and PA14 accessory expression for Taylor to use for her benchmark analysis
2. Calculate enrichment score to compare most stable genes found using the RNA-seq compendium vs the array compendium to complement in information in the correlation plot.

### CODE REVIEW

**Comment by ajlee21** [comment]
> Some questions:
> 
> * [ ]  it looks like `[0_explore_data/cluster_by_accessory_gene.ipynb](https://github.com/greenelab/core-accessory-interactome/pull/56/files#diff-a7bb150c2b1860ed71a6702423a4edc1d38d185d5d59e174d90f6e435f60cb66)` completely changed, are the results totally different  or is it not that big of a change?
> * [ ]  I don't understand why the results in [3_core_core_analysis/4_find_related_acc_genes.ipynb](https://github.com/greenelab/core-accessory-interactome/pull/56/files#diff-d12c1c8e82eca09c0c51114e36d8a4a0bf7c291410e85669fc37579b49affc70) changed

So the differences should just reflect the fact that I reran the notebooks but there isn't a change in the results comparing the previous results to this one. But perhaps you're referring to something specific that I missed. Please let me know if that's the case

File: 0_explore_data/cluster_by_accessory_gene.ipynb [hunk 1]
```diff
@@ -2024,147 +2024,147 @@
        "  <thead>\n",
        "    <tr style=\"text-align: right;\">\n",
        "      <th></th>\n",
-       "      <th>PA2004</th>\n",
```
Comment (taylorreiter) [inline 0_explore_data/cluster_by_accessory_gene.ipynb:2027]: why did these lists of genes change?

File: 0_explore_data/cluster_by_accessory_gene.ipynb [hunk 2]
```diff
@@ -2024,147 +2024,147 @@
...
-       "      <td>9.203628</td>\n",
-       "      <td>96.638095</td>\n",
-       "      <td>309.328188</td>\n",
-       "      <td>1793.988446</td>\n",
+       "      <td>720.471512</td>\n",
+       "      <td>392.592261</td>\n",
        "      <td>3.307554</td>\n",
-       "      <td>22.577650</td>\n",
-       "      <td>244.039951</td>\n",
+       "      <td>22.865264</td>\n",
```
Comment (taylorreiter) [inline 0_explore_data/cluster_by_accessory_gene.ipynb:2056]: why did these numbers change so much? Is it just an off index, and the actual result didn't change?

File: 0_explore_data/cluster_by_accessory_gene.ipynb [hunk 3]
```diff
@@ -2174,33 +2174,33 @@
        "</div>"
       ],
       "text/plain": [
-       "                PA2004      PA2174      PA1173      PA0101      PA1242  \\\n",
-       "ERX1272616   87.578274   55.796995   23.440490   63.993977   34.513605   \n",
-       "ERX1272618  223.430398  107.107103   91.663753  292.676385  256.309788   \n",
-       "ERX1272619  240.507849  114.761277   93.725605  248.220929  262.010980   \n",
-       "ERX1296067  433.829656  549.973029  316.661489  100.429857  177.972639   \n",
-       "ERX1296068  529.491131  538.185333  284.343151  141.102616  257.832963   \n",
+       "                 PA4655      PA1271     PA1705      PA3140      PA2312  \\\n",
```
Comment (taylorreiter) [inline 0_explore_data/cluster_by_accessory_gene.ipynb:2177]: why did these genes change?

File: 0_explore_data/cluster_by_accessory_gene.ipynb [hunk 4]
```diff
@@ -2265,147 +2265,147 @@
...
-       "      <th>PA14_18120</th>\n",
-       "      <th>PA14_68640</th>\n",
-       "      <th>PA14_22980</th>\n",
-       "      <th>PA14_55900</th>\n",
-       "      <th>PA14_51690</th>\n",
-       "      <th>PA14_48770</th>\n",
-       "      <th>PA14_15130</th>\n",
-       "      <th>PA14_58360</th>\n",
-       "      <th>PA14_43030</th>\n",
+       "      <th>PA14_71150</th>\n",
```
Comment (taylorreiter) [inline 0_explore_data/cluster_by_accessory_gene.ipynb:2268]: why did these gene lists change?

File: 0_explore_data/cluster_by_accessory_gene.ipynb [hunk 5]
```diff
@@ -2415,33 +2415,33 @@
        "</div>"
       ],
       "text/plain": [
-       "            PA14_23200   PA14_18120  PA14_68640    PA14_22980  PA14_55900  \\\n",
-       "ERX1272616  379.235002  7749.749646   34.701242  14533.842460   55.551148   \n",
-       "ERX1272618  660.010420   308.055613  121.801231    154.281559  469.442244   \n",
-       "ERX1272619  681.265559   318.698542  145.121657    162.194794  365.649666   \n",
-       "ERX1296067  514.657197   800.828079   52.834212   1293.139002  115.369133   \n",
-       "ERX1296068  460.820654   814.641846   80.538784   1261.436895  168.018073   \n",
+       "            PA14_71150   PA14_21210  PA14_52890  PA14_16480  PA14_29540  \\\n",
```
Comment (taylorreiter) [inline 0_explore_data/cluster_by_accessory_gene.ipynb:2418]: why did these genes change?

**Review by taylorreiter — COMMENTED** [review]
i added some inline comments where there were changes I didn't understand

File: 0_explore_data/cluster_by_accessory_gene.ipynb (hunk 3 above)
Comment (ajlee21) [reply 0_explore_data/cluster_by_accessory_gene.ipynb:2177]: I just checked that these column values are unchanged but the ordering of the columns are different, probably due to some randomness in the set operations I used to select gene ids/columns. This is the same explanation for the columns below

File: 0_explore_data/cluster_by_accessory_gene.ipynb (hunk 2 above)
Comment (ajlee21) [reply 0_explore_data/cluster_by_accessory_gene.ipynb:2056]: This ordering of this column is different but the results are the same.

**Comment by ajlee21** [comment]
Thank you for being so thorough. It does look like the results are the same, but these changed lines are marking differences in the column orderings due to randomness in the set function I used to select the columns for the dataframe. I hope that makes sense

**Review by taylorreiter — APPROVED** [review]
LGTM!

**Pull request merged**